### ISSUE TITLE
about validate

### ISSUE THREAD

**Issue by lili1052** [yastrebksv/TrackNet#7]
the tp always 0

**Comment by hhk123** [comment]
the same 
precision = 0.0
recall = 0.0
f1 = 0.0
val loss = 0.005732271736659189

**Comment by hhk123** [comment]
Correction, the final F1 score is 87% after 500 epoch of training

**Comment by TC120** [comment]
the same

**Comment by TC120** [comment]
<img width="1280" height="790" alt="Image" src="https://github.com/user-attachments/assets/0bcee08f-fc39-4d6e-9a9c-c194b9b8df10" />in my case, P&R only began to register values after approximately 60 epochs when i was training the model from scratch. I had trained the model for over 400 epochs but still had poor P&R. I am trying to improve the parameters.

<img width="376" height="73" alt="Image" src="https://github.com/user-attachments/assets/63c3917b-4849-4c34-ada4-8175b2864d55" />
